tests/apollo/test_skvbc_network_partitioning.py

```diff
@@ -47,7 +47,7 @@ class SkvbcNetworkPartitioningTest(unittest.TestCase):
 
     @with_trio
     @with_bft_network(start_replica_cmd)
-    @verify_linearizability
+    @verify_linearizability(pre_exec_enabled=False)
```

Comment:
How about simply:
`@verify_linearizability()`, in order to use the default value?

Reply:
done